src/main/java/org/jabref/gui/mergeLibraries/MergeCommand.java

```diff
@@ -0,0 +1,198 @@
+package org.jabref.gui.mergeLibraries;
+
+import javafx.collections.ObservableList;
+import org.jabref.gui.DialogService;
+import org.jabref.gui.Globals;
+import org.jabref.gui.JabRefFrame;
+import org.jabref.gui.StateManager;
+import org.jabref.gui.actions.SimpleCommand;
+import org.jabref.gui.dialogs.BackupUIManager;
+import org.jabref.gui.importer.actions.OpenDatabaseAction;
+import org.jabref.gui.shared.SharedDatabaseUIManager;
+import org.jabref.gui.util.DefaultTaskExecutor;
+import org.jabref.gui.util.DirectoryDialogConfiguration;
+
+import org.jabref.logic.autosaveandbackup.BackupManager;
+import org.jabref.logic.database.DuplicateCheck;
+import org.jabref.logic.importer.Importer;
+import org.jabref.logic.importer.OpenDatabase;
+import org.jabref.logic.importer.ParserResult;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.logic.shared.DatabaseNotSupportedException;
+import org.jabref.logic.shared.exception.InvalidDBMSConnectionPropertiesException;
+import org.jabref.logic.shared.exception.NotASharedDatabaseException;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.database.BibDatabaseMode;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BibEntryTypesManager;
+import org.jabref.preferences.PreferencesService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.jabref.logic.database.DuplicateCheck.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.lang.module.Configuration;
+import java.nio.file.*;
+import java.sql.Array;
+import java.sql.SQLException;
+import java.util.*;
+
+
+public class MergeCommand extends SimpleCommand {
```

Comment:
Can you please use a more descriptive name; maybe `MergeLibrariesInFolderIntoCurrent`?